src/main/java/bio/terra/service/filedata/flight/ingest/FileIngestFlight.java

```diff
@@ -44,6 +46,7 @@ public FileIngestFlight(FlightMap inputParameters, Object applicationContext) {
         ApplicationConfiguration appConfig =
             (ApplicationConfiguration)appContext.getBean("applicationConfiguration");
         ConfigurationService configService = (ConfigurationService)appContext.getBean("configurationService");
+        IamService iamClient = (IamService)appContext.getBean("iamService");
```

Comment:
this is the code for the single file load. We are trying to make this obsolete.
Do you really need to use this method any more?

Reply:
We haven't shifted off of it for the one project blocked by this issue, but I was also assuming the bulk load ended up delegating to this flight at some point.

Reply:
I see FileIngestWorkerFlight now, I'll update there too.

Reply:
No, processing is incompatible between the two flights. We really need to get rid of this one. We have not been maintaining it in conjunction with the other ingest flights.

Reply:
I'm +1 in general to ripping out the single-file case and migrating our one use of it, but would rather not block the IAM change on it.